core/src/test-integration/java/io/neow3j/protocol/Neow3jIntegrationTest.java

```diff
@@ -1,16 +1,18 @@
 package io.neow3j.protocol;
 
-import static io.neow3j.protocol.TestHelper.NEO_HASH;
 import static org.awaitility.Awaitility.await;
-import static org.hamcrest.Matchers.greaterThanOrEqualTo;
+import static org.hamcrest.Matchers.*;
```

Comment:
You need to configure your IDE. In our project we don't use wildcards for imports. 👍